### DISCUSSION TITLE
Ability to add table caption

### DISCUSSION THREAD

**Discussion by Yunchitz** [ueberdosis/tiptap#4921] [Feature Requests]
### Description

Wold be good to have field in editor.chain().focus().insertTable - caption

### Use Case

I want to use this feature to creating captions for table

### Type

New extension

**Comment by jfbrennan** [comment]
We need this basic table feature as well. Looking for the ability to produce standard HTML table captions like:
```
<table>
  <caption>My table description</caption>
  ...
</table>
```

**Comment by rickdoesburg** [comment]
We need this feature as well to pass WCAG Audits of our tables.

**Comment by Cirieno** [comment]
I'm currently using a terribly hacky workaround: In the TT editor I allow the user to create a \<blockquote\> with class "table-caption" (because BQ is the only container element that works here) in the first row, inside which they can have only H2/3 or P tags.  Then in my Umbraco-powered JSON I look for this blockquote and extract its content into a \<caption\> tag before serving HTML to the browser.  (One could also do this with client-side JS after serving but I wouldn't advise it.)

We really shouldn't have to dig around to make workarounds for basic semantic HTML in an HTML editor! -- while also polluting our CMS source HTML.

**Reply by rickdoesburg** [reply]
That’s quite the hack indeed. Not something particularly user friendly and indeed polluting the CMS. Besides this caption issue we’re also missing the <thead> tag availability.
